service/controller/clusterapi/v27/resource/ipam/create.go

```diff
@@ -270,40 +284,40 @@ func getVPCSubnets(ctx context.Context) ([]net.IPNet, error) {
 	return results, nil
 }
 
-// splitSubnetToStatusAZs splits subnet such that each AZ gets private and
-// public network. Size of these subnets depends on subnet.Mask and number of
-// AZs.
-func splitSubnetToStatusAZs(subnet net.IPNet, AZs []string) ([]v1alpha1.AWSConfigStatusAWSAvailabilityZone, error) {
```

Comment:
Can this actually be dropped now?

Reply:
Yes, I think so. If we need something like this, we need to rewrite it anyway for new types.

Reply:
Dropped